client/src/pages/Signup.js

```diff
@@ -7,177 +7,227 @@ import {
+                                </Label>
+                            </Col>
+                        </Row>
 
-                <Row>
-                    <Col>
-                    <Label>Email
+                        <Row>
+                            <Col xs="6">
+                                <Label>Email
                         <Input
-                            type="email"
-                            name="email"
-                            value={this.state.email}
-                            onChange={this.handleChange}
-                            />
-                    </Label>
-                    </Col>
-                </Row>
-                <Row>
-                    <Col>
-                                 <Label>Choose Your Neighborhood
+                                        type="email"
+                                        name="email"
+                                        value={this.state.email}
+                                        onChange={this.handleChange}
+                                    />
+                                </Label>
+                            </Col>
+                            <Col xs="6">
+                                <Label>Choose Your Neighborhood
 
-                <Input 
-                        value={this.state.neighborhood}
-                        onChange={this.handleChange}
-                        type="select" 
-                        name="neighborhood" >
-                        <option>Fisherman&apos;s Wharf</option>
-                        <option>North Beach</option>
-                        <option>Chinatown</option>
-                        <option>Embarcadero &#47; Financial District</option>
-                        <option>SOMA &#47; Yerba Buena</option>
-                        <option>Union Square</option>
-                        <option>Central Market</option>
-                        <option>Castro &#47; Noe Valley</option>
-                        <option>Civic Center &#47; Hayes Valley</option>
-                        <option>Haight&#45;Ashbury</option>
-                        <option>Fillmore</option>
-                        <option>Japantown</option>
-                        <option>Nob Hill</option>
-                        <option>Marina &#47; Pacific Heights</option>
-                        <option>Presidio &#47; Richmond</option>
-                        <option>Sunset</option>
-                        <option>Bayview</option>
-                        <option>Treasure Island</option>
-                        <option>Mission District</option>
+                <Input
+                                        value={this.state.neighborhood}
+                                        onChange={this.handleChange}
+                                        type="select"
+                                        name="neighborhood" >
+                                        <option>Fisherman&apos;s Wharf</option>
```

Comment:
hmmm anyway we can put this is a array and loop threw it to make react element? 🤔